tests/unit/options.js

```diff
@@ -1676,17 +1676,35 @@ exports.nonbsp = function (test) {
 /** Option `nocomma` disallows the use of comma operator. */
 exports.nocomma = function (test) {
   // By default allow comma operator
-  TestRun(test)
+  TestRun(test, "nocomma off by default")
     .test("return 2, 5;", {});
```

Comment:
something I don't get: we're not complaining about return statements being used outside of functions, which is an early error everywhere